bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/DamageDeathMove.java

```diff
@@ -475,7 +477,7 @@ public void onDeath(PlayerDeathEvent e) {
 
             // handle drops
             if (PlayerDrops.handlePlayerDrops(a, victim, killer, victimsTeam, killersTeam, cause, e.getDrops())) {
-                e.getDrops().clear();
+                e.getDrops().addAll(new ArrayList<>());
```

Comment:
Can you explain why you made this change? To me this kinda looks like a weird workaround to clear data.

Reply:
while using the clear method i was getting UnsupportedOperationException using the above code so i replaced it by providing any empty arraylist and it worked without any exceptions

Reply:
hhmmm thats kinda weird, do you have the original error still?

Reply:
yup I do have it, here https://pastebin.com/KdjGQr2N

Reply:
Just tested, it seems to call playerDeathEvent just once

Reply:
Im still not totally convinced, I will test this a bit in the next few days.